duet/views/ad_cost_breakdowns.view.lkml

```diff
@@ -0,0 +1,124 @@
+include: "//looker-hub/revenue/views/ad_cost_breakdowns.view"
+
+view: +ad_cost_breakdowns {
+
+  dimension: campaign_name {
+    sql: ${TABLE}.campaign_name ;;
+    type: string
+  }
+
+  dimension: cost_per_dou_micros {
+    sql: ${TABLE}.cost_per_dou_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_install_micros {
+    sql: ${TABLE}.cost_per_install_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_marketing_ad_click_micros {
+    sql: ${TABLE}.cost_per_marketing_ad_click_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: cost_per_revenue_generating_ad_click_micros {
+    sql: ${TABLE}.cost_per_revenue_generating_ad_click_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  dimension: ad_conversions {
+    sql: ${TABLE}.ad_conversions ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_ad_conversions {
+    label: "ad_conversions"
+    sql: ${TABLE}.ad_conversions ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: ad_impressions {
+    sql: ${TABLE}.ad_impressions ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_ad_impressions {
+    label: "ad_impressions"
+    sql: ${TABLE}.ad_impressions ;;
+    type: sum_distinct
+    sql_distinct_key: ${TABLE}.campaign_name ;;
+  }
+
+  dimension: campaign_spend_in_micros {
+    sql: ${TABLE}.campaign_spend_in_micros ;;
+    type: number
+    hidden: yes
+  }
+
+  measure: measure_campaign_spend_in_micros {
```

Comment:
These measures might be the only thing you need here (unless you're changing the dimension definitions from `looker-hub`'s versions)

Reply:
It turns out I might not need the measures at all if I'm summing on the numerical dimensions 😅 . I'm still very much figuring out the Looker tool.

Reply:
Measures are nice for common metrics, though!